idl/chromadb/proto/chroma.proto

```diff
@@ -93,6 +93,11 @@ message OperationRecord {
     Operation operation = 4;
 }
 
+message RequestVersionContext {
+    uint32 collection_version = 1;
+    uint64 log_position = 2;
```

Comment:
nit: log_offset is the term used everywhere else. Don't have any strong preference on which is better but should we have a consistent naming everywhere?

Reply:
We should. But this inconsistency is not introduced here (i.e "name used everywhere else" is not true"). 

If you look at message Collection {} for example 

```
message Collection {
  string id = 1;
  string name = 2;
  string configuration_json_str = 3;
  optional UpdateMetadata metadata = 4;
  optional int32 dimension = 5;
  string tenant = 6;
  string database = 7;
  int64 log_position = 8;
  int32 version = 9;
}
```

It is log_position. I went off what the collection called it, I don't know how to choose haha.